src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -41,21 +41,28 @@ public class EditCommand extends Command {
     public static final String COMMAND_WORD = "edit";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
-            + "by the index number used in the displayed person list. "
+            + "by the index number used in the displayed person list. \n"
```

Comment:
i think this one should be by the person's ic number?

Reply:
okay fixed, let me know if there are other typos